### DISCUSSION TITLE
No microphone support passthrough in host to client (like it's implemented with audio sound output)

### DISCUSSION THREAD

**Discussion by ushabuntu** [rustdesk/rustdesk#6868] [Feature Request]
When connected to the host device - playing audio in host works fine and plays audio in connected client
This is not the case with microphone, and it's very useful - e.g. when connecting to client and using VOIP or Teams / Zoom calls 
The feature - when working as intended - will allow client to use microphone and passthrough the mic output to the host machine (as in sound / audio input)

**Comment by Mathis-Aubry** [comment]
We need this, put some attention here !
Maybe you'll need to rephrase, your explanation seem a bit unclear

**Reply by pdifeo** [reply]
Same question from me

https://github.com/rustdesk/rustdesk/discussions/9219

**Comment by yen360** [comment]
Bump!
